docs/core/preview3/tools/csproj.md

```diff
@@ -136,34 +197,91 @@ The following example specifies the fallbacks only for the `netcoreapp1.0` targe
+
+### IsPackable
+TBD
+
+### PackageVersion
+Specified a version that the resulting package will have. Accepts all forms of NuGet version string. 
+
+### PackageId
+Specify a name for the resulting package. If not specified, the `pack` operation will default to using the AssemblyName or directory name as the name of the package. 
+
+### Title
+A human-friendly title of the package, typically used in UI displays as on nuget.org and the Package Manager in Visual Studio. If not specified, the package ID is used instead.
+
+### Authors
+A comma-separated list of packages authors, matching the profile names on nuget.org. These are displayed in the NuGet Gallery on nuget.org and are used to cross-reference packages by the same authors.
+
+### Description
+A long description of the package for UI display.
+
+### Copyright
+Copyright details for the package.
+
+### PackageRequireLicenseAcceptance
+A Boolean value specifying whether the client must prompt the consumer to accept the package license before installing the package.
+
+### PackageLicenseUrl
+An URL to the license that is applicable to the package.
+
+### PackageProjectUrl
+A URL for the package's home page, often shown in UI displays as well as nuget.org.
+
+### PackageIconUrl
+A URL for a 64x64 image with transparenty background to use as the icon for the package in UI display.
+
+### PackageReleaseNotes
+Release notes for the package.
+
+### PackageTags
+A list of tags that designates the package. The list is represented as a list of tags that are comma-delimited. 
+
+### PackageOutputPath
+TBD
+
+### IncludeSymbols
+This boolean value indicates whether the package should include symbols when it is packed. The default value is "false". 
+
+### IncludeSource
+This boolean value indicates whether the packing process should include source code in the resulting NuGet package. 
+
+### PackageTypes
+TBD
+
+### IsTool
+Specifies whether all output files, as specified in the Output Assemblies scenario, are copied to the tools folder instead of the lib folder. Note that this is different from a DotNetCliTool which is specified by setting the PackageType in csproj file.
+
+### RepositoryUrl
+Specifies the URL for the repository where the source code for the package resides and/or from which it is being built. 
+
+### RepositoryType
+Specifies the type of the repository. Default is "git". 
+
+### NoPackageAnalysis
+TBD
+
+### MinClientVersion
+Specifies the minimum version of the NuGet client that can install this package, enforced by nuget.exe and the Visual Studio Package Manager.
+
+### IncludeBuildOutput
+This is a boolean, which decided whether the build output assemblies should be packed into the nupkg or not.
+
+### IncludeContentInPack
+TBD
+
+### BuildOutputTargetFolder
+Specify the folder in which the output assemblies should go to. The output assemblies (and other output files) are copied into their respective framework folders.
+
+### ContentTargetFolders
+TBD
```

Comment:
by default, the value is "content;contentFiles" . These specify the default location of where all the files should go if PacakgePath is not specified.